src/main/java/com/synopsys/integration/alert/ApplicationConfiguration.java

```diff
@@ -136,4 +138,23 @@ public AuthorizationManager authorizationManager(RoleAccessor roleAccessor) {
         return new AuthorizationManager(roleAccessor);
     }
 
+    @Bean
+    public DefaultJmsListenerContainerFactory defaultJmsListenerContainerFactory(CachingConnectionFactory connectionFactory) {
+        DefaultJmsListenerContainerFactory factory = new DefaultJmsListenerContainerFactory();
+        factory.setConnectionFactory(connectionFactory);
+        return factory;
+    }
+
+    @Bean
```

Comment:
Can we register these beans in their own configuration class? e.g. something called `AlertJmsConfiguration`.